File: app/routes/me.py
from fastapi import APIRouter, Depends
from ..auth_dep import get_current_user
from ..models import User

router = APIRouter(prefix="/api/me", tags=["me"])

@router.get("")
def me(user: User = Depends(get_current_user)):
    return {
        "user": {
            "id": user.id,
            "tgId": str(user.tg_id),
            "username": user.username,
            "firstName": user.first_name,
            "lastName": user.last_name,
            "photoUrl": user.photo_url,
        }
    }

File: apps/api/src/routes/me.ts
import { Router } from "express";
import { authMiddleware, type AuthedRequest } from "../middleware/auth";
import { prisma } from "../lib/prisma";

export const meRouter = Router();

meRouter.get("/", authMiddleware, async (req, res) => {
  const { userId } = (req as AuthedRequest).auth;
  const user = await prisma.user.findUnique({ where: { id: userId } });
  if (!user) return res.status(404).json({ error: "NOT_FOUND" });

  return res.json({
    user: {
      id: user.id,
      tgId: String(user.tgId),
      username: user.username,
      firstName: user.firstName,
      lastName: user.lastName,
      photoUrl: user.photoUrl
    }
  });
});
